.travis/after_script.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -x
```

Comment:
This line should probably go after the header.

Reply:
Good spot! Fixing it. Thanks!